### PR TITLE
Extending v2

### PR DESCRIPTION
**Pull request by RaphaelS1**

### CODE REVIEW

**Review by RaphaelS1 — COMMENTED** [review]
@be-marc would you be able to move [this](https://github.com/mlr-org/mlr3book/pull/543/commits/13a94baef6232df501f277ca5c53438865f15e85) into a vignette in mlr3tuning?

**Comment by RaphaelS1** [comment]
@mb706 would you be able to move [this](https://github.com/mlr-org/mlr3book/pull/543/commits/54461c67c6a137220726c8092f203ae985b6a44a) into a vignette in mlr3pipelines, which I'll then link to from the book? Definitely doesn't need to be perfect but just need it to be somewhere.

**Comment by sebffischer** [comment]
This is a summary comment with TODOs: 

* [ ] Port the "extending tuning" section to mlr3tuning. I started a PR [here](https://github.com/mlr-org/mlr3tuning/pull/386)
* [ ] Port the "extending pipelines" section to mlr3pipelines. I started a PR [here](https://github.com/mlr-org/mlr3pipelines/pull/716)

**Comment by sebffischer** [comment]
@RaphaelS1 if you have time please iterate once more :)

**Review by RaphaelS1 — COMMENTED** [review]
Can you uncommit [this](https://github.com/mlr-org/mlr3book/pull/543/files#diff-97b46a68931fcdbc4634f120388ccccd0a52f1af91b42d45dfe2802d3d7ce844)? We don't want to merge by mistake

File: book/extending.qmd [hunk 1]
```diff
@@ -1,41 +1,46 @@
...
+Or if you are considering adding a new machine learning task then please contact us on GitHub, email, or Mattermost.
+
+To understand this chapter, some basic familiarity with the `r ref_pkg("R6")` class system is required.
+@sec-r6 contained a brief introduction and references to further resources.
+This chapter does not contain information about e.g. coding-style, contribution guides, or other developer related information.
+Such information can be found in the `r link("https://github.com/mlr-org/mlr3/wiki", "mlr3 Wiki")`.
 
 We welcome contributions from all levels of developers and if you want to add any of your new classes to our universe then please make pull requests to aligning packages, for example tuners and filters would go to `r mlr3tuning` and `r mlr3filters` respectively, a new survival measure would go to `r mlr3proba`, and *all* new learners go to `r mlr3extralearners`.
 Do not worry if you make a PR to the wrong repository, we will transfer it to the right one.
```
Comment (RaphaelS1) [inline book/extending.qmd:37]:
```suggestion
Do not worry if you make a PR to the wrong repository, we will transfer it to the right one.
Please read the `r link("https://github.com/mlr-org/mlr3/wiki", "mlr3 Wiki")` for coding conventions that we use if you want to add code to our organisation. 
```

**Pull request closed without merging**